docs/qiita-16S-analysis.rst

```diff
@@ -3,218 +3,424 @@
+   :align:   center
+
+Under "Select a Shape Category" you can alter the shape of each group on the PCoA plot to the following:
+
+.. figure::  images/shape_options.png
+   :align:   center
+
+Under the "Axis" tab you will notice 5 pull-down menus:
+
+.. figure::  images/axis_tab.png
+   :align:   center
+
+The first 3 pull-down menus located under "Visible" allow you to change the axis that are being displayed.
+The "Axis and Labels Color" pull-down menu allow you to change the color of your axis and label of the PCoA.
+The "Background Color" pull-down menu allows you to change the color of the background of the PCoA.
+The % Variantion Expanded graph displays how different the most dissimilar samples are by percentage for each axis that can be used. 
+
+Under the "Scale" tab you will notice 2 pull-down menus:
+
+.. figure::  images/scale_tab.png
+   :align:   center
+
+Under "Select a Scale Category" you can choose the grouping of your samples. Under "Global Scaling" you can change the point size for each group on the PCoA plot. 
+
+Here, I’ve colored the points in our test data by cat ownership.
+
+.. figure::  images/pcoa.png
+   :align:   center
+
+Let’s take a few minutes now to explore the various features of Emperor. Open a new browser window with the 'Emperor tutorial <https://biocore.github.io/emperor/tutorial_index.html#section1>'__ and follow along with your test data.
+
+Another way to study the beta diversity is by measuring the beta diversity group significance. Beta diversity group significance measures whether groups of samples are significantly different from one another using a permutation-based statistical test.
+
+To perform a beta group significance analysis, select the rarefied "dflt_name - BIOM" artifact in the processing network and select "Process". Then select the "dflt_name - BIOM" artifact and select "beta_diversity" from the drop-down menu. The parameters will appear below the workflow diagram: 
+
+.. figure::  images/beta_group_significance_beta.png
+   :align:   center
+
+Several parameter have automatically selected for you. In the "p-metric" field enter "unweighted Unifrac" and in the "i-tree" field enter "/databases/gg/13_8/trees/97_otus.tree", and click "Add Command".
+
+To create the beta group significance analysis, select select "beta_group_significance" from the drop-down menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/significance_matrix.png
+   :align:   center
+
+All of the parameter have automatically selected for you just click "Add Command". Once the command is added the workflow should appear as follows:
+
+.. figure::  images/beta_group_significance_workflow.png
+   :align:   center
+
+Once the q2 visualization artifact is chosen in the network, the beta diversity box plots will appear: 
+
+.. figure::  images/beta_significance_boxplot.png
+   :align:   center
+
+The PERMANOVA (Permutational multivariate analysis of variance) results will also be displayed:
```

Comment:
The PERMANOVA `(Permutational multivariate analysis of variance) <http://onlinelibrary.wiley.com/doi/10.1111/j.1442-9993.2001.01070.pp.x/full>`__ test results...